drivers/Kconfig

```diff
@@ -0,0 +1,9 @@
+# Copyright (c) 2020 HAW Hamburg
+#
+# This file is subject to the terms and conditions of the GNU Lesser
+# General Public License v2.1. See the file LICENSE in the top level
+# directory for more details.
+
+menu "Drivers"
+source "$(RIOTCPU)/nrf52/radio/nrf802154/Kconfig"
```

Comment:
We could add categories similar to what we already have [in the docs](http://doc.riot-os.org/group__drivers.html).

Reply:
done